### DISCUSSION TITLE
Development branch

### DISCUSSION THREAD

**Discussion by petter1** [johannrichard/homebridge-dingz#541] [General]
Would it make sense to create a development branch, so that I can see where the development is, and maybe contribute via an own branch and small pull requests?

**Comment by johannrichard** [comment]
Sorry haven't seen the comment until now. Yes, that could definitely make sense. I'll set one up.

**Comment by johannrichard** [comment]
Please have a look at the (brief) guidelines I wrote:

[CONTRIBUTING.md](https://github.com/johannrichard/homebridge-dingz/blob/next/CONTRIBUTING.md). Happy coding. 😉
